lib/rules/jsx-indent-props.js

```diff
@@ -140,6 +141,7 @@ module.exports = {
      * @return {Number} Indent
      */
     function getNodeIndent(node) {
+      // @ts-ignore
```

Comment:
ts-expect-error

Reply:
Thanks! Fixed in [eb44ba7](https://github.com/jsx-eslint/eslint-plugin-react/pull/3731/commits/eb44ba71aba5ed0d36518b3c658b7cab5e59c14f) .